src/components/posts/AddPostForm.js

```diff
@@ -0,0 +1,58 @@
+import { useRef } from "react";
+import classes from "./AddPostForm.module.css";
+
+function AddPostForm(props) {
+  const authorInputRef = useRef();
+  // const dateInputRef = useRef();
+  const avatarInputRef = useRef();
+  const contentInputRef = useRef();
+
+  function submitHandler(event) {
+    event.preventDefault();
+    const enteredAuthor = authorInputRef.current.value;
+    // const enteredDate = dateInputRef.current.value;
+    const enteredDate = new Date().toISOString().slice(0, 10);
+    const enteredAvatar = avatarInputRef.current.value;
+    const enteredContent = contentInputRef.current.value;
+
+    authorInputRef.current.value = "";
+    // dateInputRef.current.value = "";
+    avatarInputRef.current.value = "#000000";
+    contentInputRef.current.value = "";
+
+    const postData = {
+      author: enteredAuthor,
+      avatar: enteredAvatar,
+      date: enteredDate,
+      content: enteredContent,
+      likes: 0
+    };
+
+    props.onAddPost(postData);
+  }
+  return (
+    <form className={classes.form} onSubmit={submitHandler}>
+      <div className={classes.control}>
+        <label htmlFor="author">Author: </label>
+        <input type="text" required id="author" ref={authorInputRef} />
+      </div>
+      {/* <div className={classes.control}>
+        <label htmlFor="date">Date: </label>
+        <input type="date" required id="date" ref={dateInputRef} />
+      </div> */}
```

Comment:
When doing multi-line comments, you always want to individually comment every line. To uncomment this you would only be changing two lines but it would be affecting four which is deceptive. The main issue is that if someone else was also working on the codebase and committed something it could get caught by you commenting in a merge and that is a nightmare to spot and fix

Reply:
Yeah that makes a lot of sense, will try to make sure I do that